src/gui/executionlogwidget.cpp

```diff
@@ -66,42 +61,63 @@ ExecutionLogWidget::~ExecutionLogWidget()
     delete m_ui;
 }
 
-void ExecutionLogWidget::showMsgTypes(const Log::MsgTypes &types)
+void ExecutionLogWidget::showMsgTypes(const Log::MsgTypes types)
 {
-    m_msgList->showMsgTypes(types);
+    m_msgList->setMsgTypes(types);
 }
 
 void ExecutionLogWidget::addLogMessage(const Log::Msg &msg)
 {
-    QString colorName;
-    switch (msg.type) {
-    case Log::INFO:
-        colorName = QLatin1String("blue");
-        break;
-    case Log::WARNING:
-        colorName = QLatin1String("orange");
-        break;
-    case Log::CRITICAL:
-        colorName = QLatin1String("red");
-        break;
-    default:
-        colorName = QApplication::palette().color(QPalette::WindowText).name();
-    }
-
-    const QDateTime time = QDateTime::fromMSecsSinceEpoch(msg.timestamp);
-    const QString text = QString(QLatin1String("<font color='grey'>%1</font> - <font color='%2'>%3</font>"))
-        .arg(time.toString(Qt::SystemLocaleShortDate), colorName, msg.message);
+    QDateTime time = QDateTime::fromMSecsSinceEpoch(msg.timestamp);
+    QString text = QString("%1 - %2").arg(time.toString(Qt::SystemLocaleShortDate), msg.message);
```

Comment:
Should I keep nagging about `const`? honest question.
I would add them myself, but I find it a bit disturbing to keep mentioning it...

Reply:
Sorry. Now I think I got all of them. If not, don't hesitate to leave a comment.